.github/workflows/win.yml

```diff
@@ -86,6 +86,18 @@ jobs:
         run: |
           cmake . -DCMAKE_POLICY_DEFAULT_CMP0091=NEW -DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded -DCMAKE_FIND_USE_SYSTEM_ENVIRONMENT_PATH=OFF
           cmake --build . --config Release
+      - name: Download mpir
+        if: steps.cache-libs.outputs.cache-hit != 'true'
+        uses: actions/checkout@v2
+        with:
+          repository: BrianGladman/mpir
+          ref: d9c9a842be6475bef74324f367ce2c5a78c55d06  # master @ 2021-10-14
+          path: mpir
+      - name: Build mpir
+        if: steps.cache-libs.outputs.cache-hit != 'true'
+        working-directory: ./mpir
+        run: |
+          MSBuild.exe /p:Platform=x64 /p:Configuration=Release /p:DefineConstants=MSC_BUILD_DLL .\msvc\vs19\lib_mpir_gc\lib_mpir_gc.vcxproj
```

Comment:
Hm, this "vs19" part could turn out not future-proof. Because we never explicitly choose that version of MSVC. Would a wildcard in its place work?

Reply:
Since VSWhere is available on the runners, perhaps we could do:

```powershell
"vs$((vswhere -property displayName) -replace '.*\b\d\d(\d\d)\b.*', '$1')"
```